src/e-init.js

```diff
@@ -145,10 +129,15 @@ program
   .option('--lsan', `When building, enable clang's leak sanitizer`, false)
   .option('--bootstrap', 'Run `e sync` and `e build` after creating the build config.')
   .option(
-    '--use-goma',
-    `Use Electron's custom deployment of Goma (only available to maintainers).`,
+    '--disable-goma',
+    'Disables usage of Goma, this will make the build take substantially longer',
     false,
   )
+  .option(
+    '--goma <target>',
+    `Use Electron's custom deployment of Goma.  Can be "cache-only" or "cluster".  The "cluster" mode is only available to maintainers`,
+    'cache-only',
+  )
```

Comment:
These two options seem to overlap.

What would you think of having three targets: `cluster`, `cache-only`, or `none`, and removing the `--disable-goma` option?